packages/react-composites/src/composites/CallWithChatComposite/MobilePane.tsx

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { concatStyleSets, DefaultButton, FontIcon, Stack } from '@fluentui/react';
+import { useTheme } from '@internal/react-components';
+import React from 'react';
+import {
+  paneBodyContainer,
+  scrollableContainer,
+  scrollableContainerContents
+} from '../common/styles/ParticipantContainer.styles';
+import { useCallWithChatCompositeStrings } from './hooks/useCallWithChatCompositeStrings';
+import {
+  hiddenMobilePaneStyle,
+  mobilePaneBackButtonStyles,
+  mobilePaneButtonStyles,
+  mobilePaneControlBarStyle,
+  mobilePaneStyle
+} from './styles/MobilePane.styles';
+
+export const MobilePane = (props: {
+  onClose: () => void;
+  onChatButtonClicked: () => void;
+  onPeopleButtonClicked: () => void;
+  children: React.ReactNode;
+  hidden: boolean;
+  dataUiId: string;
+  activeTab: MobilePaneTab;
+}): JSX.Element => {
+  // We hide the mobile pane instead of not rendering the entire pane to persist certain elements
+  // between renders. An example of this is composing a chat message - a chat message that has been
+  // typed but not sent should not be lost if the side panel is closed and then reopened.
+  const mobilePaneStyles = props.hidden ? hiddenMobilePaneStyle : mobilePaneStyle;
+  const theme = useTheme();
+  const chatAndPeoplePaneButtonStylesThemed = concatStyleSets(mobilePaneButtonStyles, {
```

Comment:
nit: Shorter name is equally good here: `buttonStyles`

Reply:
Also, since this is a `styles` object directly passed down (not via `mergeStyles`), consider `useMemo` to avoid spuriously rerendring children.